front/public/locales/de/operationalStudies/scenario.json

```diff
@@ -0,0 +1,117 @@
+{
+  "editScenario": "Szenario bearbeiten",
+  "description": {
+    "electricalProfileWithName": "Bahnstromprofile: {{name}} | ID{{id}}",
+    "electricalProfileWithId": "Bahnstromprofile: ID{{id}}"
+  },
+  "displayTrainsWithDetails": "Zugdetails anzeigen/verbergen",
+  "electricalProfileSet": "Bahnstromprofil-Satz",
+  "errorMessages": {
+    "error": "Ein Fehler ist aufgetreten",
+    "hardErrorInfra": "Das Netz enthält Fehler und kann ohne vorherige Korrektur nicht benutzt werden.",
+    "softErrorInfra": "Die Anwendung ist nicht verfügbar, ein Serverfehler ist aufgetreten, bitte versuchen Sie es später erneut.",
+    "unableToCreateScenarioMessage": "Szenario erstellen nicht möglich.",
+    "unableToLoadInfra": "Ladefehler, Neustart in Aufbau ({{reloadCount}})",
+    "unableToUpdateScenarioMessage": "Szenario aktualisieren nicht möglich.",
+    "unableToDeleteScenarioMessage": "Szenario löschen nicht möglich.",
+    "unableToDuplicateATrain": "Zug duplizieren nicht möglich."
+  },
+  "filterPlaceholder": "Filter, Tags",
+  "advancedFiltersPlaceholder": "Filtern nach Baureihe, Serie, Detail",
+  "infraLoadingState": "{{step}}/3",
+  "infrastructure": "Netz",
+  "lessDetails": "Weniger Details",
+  "macroscopic": "MAKRO",
+  "microscopic": "MIKRO",
+  "modifyScenario": "Szenario bearbeiten",
+  "moreDetails": "Mehr Details",
+  "noElectricalProfileSet": "Kein Bahnstromprofil",
+  "pacedTrain": {
+    "pacedTrain": "Taktleistung"
+  },
+  "pacedTrain_one": "1 Dienstleistung",
+  "pacedTrain_other": "{{count}} Dienstleistungen",
+  "pacedTrain_zero": "Keine Dienstleistung",
+  "pacedTrainCount_one": "1/$t(pacedTrain, { 'count': {{totalCount}} }) selected",
+  "pacedTrainCount_other": "{{count}}/$t(pacedTrain, { 'count': {{totalCount}} }) selected",
+  "pacedTrainCount_zero": "$t(pacedTrain, { 'count': {{totalCount}} })",
+  "pacedTrainAndTrainCount": "{{pacedTrainCount}}/$t(pacedTrain, { 'count': {{totalPacedTrainCount}} }) and {{trainCount}}/$t(train, { 'count': {{totalTrainScheduleCount}} }) selected",
+  "scenarioCancel": "Abbrechen",
+  "scenarioCreateButton": "Szenario erstellen",
+  "scenarioCreationTitle": "Szenario erstellen",
+  "scenarioDeleteButton": "Löschen",
+  "scenarioDeleted": "Löschen erfolgreich!",
+  "scenarioDeletedDetails": "Szenario {{ name }} ist gelöscht worden.",
+  "scenarioDescription": "Beschreibung",
+  "scenarioDescriptionInvalid": "Die Beschreibung darf 1.024 Zeichen nicht überschreiten.",
+  "scenarioDescriptionPlaceholder": "Kurzbeschreibung des Szenarios",
+  "scenarioModificationTitle": "Szenario bearbeiten",
+  "scenarioModifyButton": "Sichern",
+  "scenarioName": "Szenarienname",
+  "scenarioNameInvalid": "Der Szenarienname darf 128 Zeichen nicht überschreiten.",
+  "scenarioTags": "Tags",
+  "scenarioUpdated": "Update erfolgreich!",
```

Comment:
Same comment as earlier, maybe Aktualisierung to have the same world as the next sentence?